.pipelines/v2/templates/job-build-project.yml

```diff
@@ -499,16 +477,6 @@ jobs:
           batchSignPolicyFile: '$(build.sourcesdirectory)\.pipelines\ESRPSigning_DSC.json'
           ciPolicyFile: '$(build.sourcesdirectory)\.pipelines\CIPolicy.xml'
 
-    - template: steps-esrp-signing.yml
-      parameters:
-        displayName: Sign x86 DirectShow VCM
-        signingIdentity: ${{ parameters.signingIdentity }}
-        inputs:
-          FolderPath: 'x86/$(BuildConfiguration)' # Video conf uses x86 and x64.
```

Comment:
There's one more comment `# Video conf uses x86 and x64.` in this file, I guess just remove it so we don't have a reference to removed stuff.

Reply:
Yeah, that comment is weird... thanks for noticing.